mongo/integration/collection_test.go

```diff
@@ -1973,3 +1974,23 @@ func assertKillCursorsCommandsAreMonitored(mt *mtest.T, cmdName string, cursorFn
 	evt = mt.GetStartedEvent()
 	assert.Equal(mt, "killCursors", evt.CommandName, "expected command 'killCursors', got %q", evt.CommandName)
 }
+
+func TestSecurityToken(t *testing.T) {
+	mt := mtest.New(t)
+
+	opts := mtest.NewOptions().ClientType(mtest.Proxy)
+	mt.RunOpts("", opts, func(mt *mtest.T) {
+		_, err := mt.Coll.DeleteOne(context.Background(), bson.M{})
+		// assert.NoError(mt, err)
+		fmt.Println(err)
+
+		msgs := mt.GetProxiedMessages()
+		for _, msg := range msgs {
+			if msg.CommandName != "delete" {
+				continue
+			}
+			fmt.Printf("%q\n", msg.Sent.RawMessage)
+			fmt.Println(msg.Sent.SecurityToken)
```

Comment:
We should add assertions on the security token.

Reply:
I removed that test.